Wikipedia/Code/MWKImage.h

```diff
@@ -46,6 +46,22 @@
 
 
 /**
+ * Width of the original image file. For example, if this image is a thumbnail, this is the width of the original image.
+ */
+@property (copy, nonatomic) NSNumber* originalFileWidth;
```

Comment:
The answer now is yes we do use these new properties and it's OK for older serialized MWKImages to not have this information.